src/strategies/lodestar-aelin-vesting/index.ts

```diff
@@ -0,0 +1,35 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import { getAddress } from '@ethersproject/address';
+
+export const author = '0xAppo';
+export const version = '0.1.0';
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) =>
+    multi.call(address, options.aelinDealTokenAddress, 'balanceOf', [address])
+  );
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, balance]) => [
+      getAddress(address),
+      parseFloat(formatUnits(balance, options.decimals)) * options.aelinExchangeRate
```

Comment:
Hi @0xAppo I think you can use `math` strategy to multiply instead of a separate strategy. can try it? :)